index.html

```diff
@@ -4,7 +4,7 @@
 	<body>
 		<h1>A headline!</h1>
 		<p>Here's a first paragraph</p>
-		<p>a bug!</p>
+		<b>Heavy text</b>
```

Comment:
I think this should be a `<strong>` tag instead.
```suggestion
		<strong>Heavy text</strong>
```